fm.reaper.Reaper.yml

```
@@ -0,0 +1,55 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --env=ALSA_CONFIG_PATH=
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '21.08'
+    add-ld-path: lib
+    merge-dirs: lv2;vst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: "."
+    autodelete: false
+
+modules:
+  - name: reaper
+    buildsystem: simple
+    build-commands:
+      - sh install-reaper.sh --quiet --install /app
+      - install -Dm644 /app/REAPER/Resources/main.png /app/share/icons/hicolor/256x256/apps/fm.reaper.Reaper.png 
+      - install -Dm644 fm.reaper.Reaper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+      - mkdir -p /app/bin
+      - ln -sf /app/REAPER/reaper /app/bin/reaper 
+    sources:
+      - type: archive
+        url: https://www.reaper.fm/files/6.x/reaper658_linux_x86_64.tar.xz
+        sha256: 8d18e110a75f99640161c852acf3610dfeb2a2712a49df2012888236e89dcac3
+      - type: file
+        path: fm.reaper.Reaper.desktop
+      - type: file
+        path: fm.reaper.Reaper.metainfo.xml
+    post-install:
+      - install -d /app/extensions/Plugins
+      - mkdir -p /app/lib/ffmpeg
```

```suggestion
      - install -d /app/lib/ffmpeg
```

just to be consistent.